tests/tests/busola-login-failed.spec.js

```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress" />
+import config from '../config';
+
+const ADDRESS = config.localDev
+  ? `http://localhost:4200`
+  : `https://busola.${config.domain}`;
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // returning false here prevents Cypress from failing the test
+  debugger;
+  return false;
+});
+
+context('Busola Login Failed', () => {
+  before(() => {
+    cy.clearSessionStorage().clearLocalStorage();
+  });
+
+  it('Use wrong kubeconfig', () => {
+    cy.visit(ADDRESS)
+      .get('#textarea-kubeconfig')
```

Comment:
Please use selectors which determine what the user can see. He/she cannot see the ID of the element :) 
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements


This comment applies to almost evry selector in your code.